bedrock/mozorg/templates/mozorg/about/governance/policies/commit.html

```diff
@@ -25,7 +25,7 @@ <h1 class="mzp-c-article-title">{{ self.page_title() }}</h1>
 
 <h2 id="procedure">Procedure</h2>
 
-<p>Here is a list of the steps that need to happen to become a Mozilla committer. Employment with any particular entity (including the Mozilla Foundation or Corporation) does not change the need to follow these steps.</p>
+<p>Here is a list of the steps that need to happen to become a Mozilla committer. Employment with any particular entity (including Mozilla Foundation or Corporation) does not change the need to follow these steps.</p>
```

Comment:
Wondering whether even cosmetic changes to some of the docs, like governance, esp. here commit access policies, won't need approvals (here module ownership and peership still applies IIUC…)

Reply:
(The same applies for security/bounty docs, basically just pinging some reviewers/"codeowners" of these changes in past so they OK it, as that's their surface?)